mobile/src/main/java/org/openhab/habdroid/core/OpenHABTracker.java

```diff
@@ -95,7 +95,7 @@ public void start() {
                     // If remote URL is configured
                     if (mOpenHABUrl.length() > 0) {
                         Log.d(TAG, "Connecting to remote URL " + mOpenHABUrl);
-                        openHABTracked(mOpenHABUrl, mCtx.getString(R.string.info_conn_rem_url));
```

Comment:
I agree with the voice recognizer , my guess is that it is a very rarely used feature and probably extra noise, but I don't have an issue with the connecting messages.  I don't think I have heard others complain about it so before we make this kind of change I would prefer we have a chorus of feedback saying this is undesirable.  I actually prefer to know if I'm connecting locally or remotely.